pkg/sql/flowinfra/flow_registry.go

```diff
@@ -28,7 +28,15 @@ import (
 	"github.com/cockroachdb/redact"
 )
 
-var errNoInboundStreamConnection = errors.New("no inbound stream connection")
+// ErrNoInboundStreamConnection is the error propagated through the flow when
+// the timeout to setup the flow is exceeded.
+var ErrNoInboundStreamConnection = errors.New("no inbound stream connection")
+
+// IsNoInboundStreamConnectionError returns true if err's Cause is an
+// ErrNoInboundStreamConnection.
+func IsNoInboundStreamConnectionError(err error) bool {
+	return errors.Is(err, ErrNoInboundStreamConnection)
+}
```

Comment:
It's strange to have both the error exported as well as a IsError function to test.
I think it's one or the other: either keep the ErrNoInboundStreamConnection unexported (and keep Is method), or just export Err...

Reply:
Done.